docs/guides/isolated_systems.md

```diff
@@ -0,0 +1,203 @@
+
+We now describe the SQLMesh project workflow with isolated systems.
+
+The primary difference from the single-system workflows described above is that some steps now occur in the nonproduction system and others in the production system.
+
+In some isolated system implementations, the data in the non-production system is different from that in the production system. For example, columns containing sensitive information may be masked or altered in the non-production system's data. These potential differences increase the likelihood of unintentional errors, so we describe (and recommend) a workflow with staging environments.
+
+In the steps below, we assume that the non-production system is the default gateway. The isolated system workflow with staging environments begins like the [staging workflow above](#staging-environment), but with additional staging environment creation and validation in the production system after validation in the default non-production system:
+
+1. [Same as [above](#staging-environment)]
+
+2. [Same as [above](#staging-environment)]
+
+3. Deploy to production system `staging` environment
+
+    a. Execute the `sqlmesh --gateway production plan staging` command to apply the updates to the production system `staging` environment. Note that your production system's gateway may have a different name and that you may choose any environment name other than `prod`.
+
+    b. Developers validate the set of code updates in the production system `staging` environment
+
+4. Deploy to production
+
+    a. Execute the `sqlmesh  --gateway production plan` command to apply the updates to the production system's `prod` environment
+
+![Isolated systems SQLMesh workflow with staging environments](./isolated_systems/isolated-systems_isolated-systems-workflow.png)
+
+The classification of changes as breaking/non-breaking in the non-production system plan (step 2) will not be available to the production system plan (step 3) because the systems do not share SQLMesh state data. Therefore, the classification must occur again in the production system.
+
+In isolated systems, SQLMesh's virtual data environments will operate normally *within* each system, but not across systems. For example, when the non-production system's staging environment is created (step 2c), any computations previously run in a developer's personal environment will not run again.
+
+In step (3) above, the creation of the `staging` environment in the production system will require running all the computations required by code changes. Deployment to the production system's `prod` environment (step 4) will not require additional computation due to SQLMesh's virtual data environments. The deployment only requires updates views so will be very rapid.
+
+## SQLMesh features
+
+This section describes how two key SQLMesh features perform when a project runs on isolated systems: virtual data environments and computing only what's necessary.
+
+### Virtual data environments
```

Comment:
Do we already have a doc page for VDEs that we could reuse? This seems something which may be better off documented in its own page and then referenced here, as it's a core SQLMesh concept.